scripts/build.ts

```diff
@@ -1,28 +1,34 @@
 import path from "node:path";
-import url from "node:url";
-import fs from "node:fs/promises";
 
-import tsup from "tsup";
+import { build, LibraryFormats } from "vite";
 
-const __dirname = path.dirname(url.fileURLToPath(import.meta.url));
+const fileName = (format: string) => `index.${format === "es" ? "js" : "cjs"}`;
+const formats: LibraryFormats[] = ["es", "cjs"];
 
-const dist = path.join(__dirname, "..", "dist");
-
-await fs
-  .access(dist)
-  .then(() => fs.rm(dist, { recursive: true }))
-  .catch(() => {});
```

Comment:
Vite does cleanup, so I removed this. I will note that vite only cleans the output directory though which I've got configured to be `dist/client` and `dist/node`. Seemed like that was good enough.